configs/3.10/post_extract.patch

```diff
@@ -0,0 +1,309 @@
++	__mod_register_sched_domain_sysctl();
+ 
+ 	plugsched_cpuset_unlock();
+ 	mutex_unlock(&cgroup_mutex);
+@@ -280,8 +311,7 @@ static inline void restore_sched_domain_sysctl(void)
+ 	mutex_lock(&cgroup_mutex);
+ 	plugsched_cpuset_lock();
+ 
+-	unregister_sched_domain_sysctl();
+-	cpumask_copy(sd_sysctl_cpus, cpu_possible_mask);
++	__mod_unregister_sched_domain_sysctl();
+ 	__orig_register_sched_domain_sysctl();
+ 
+ 	plugsched_cpuset_unlock();
+@@ -307,7 +337,7 @@ void install_sched_debugfs(void)
+ }
+ 
+ extern struct file_operations __orig_sched_feat_fops;
+-extern struct seq_operations  __orig_sched_debug_sops;
++extern struct file_operations __orig_sched_debug_fops;
+ 
+ void restore_sched_debugfs(void)
+ {
+@@ -320,7 +350,7 @@ int install_sched_debug_procfs(void)
+ {
+ 	remove_proc_entry("sched_debug", NULL);
+ 
+-	if (!proc_create_seq("sched_debug", 0444, NULL, &__mod_sched_debug_sops))
++	if (!proc_create("sched_debug", 0444, NULL, &__mod_sched_debug_fops))
+ 		return -ENOMEM;
+ 
+ 	return 0;
+@@ -330,7 +360,7 @@ int restore_sched_debug_procfs(void)
+ {
+ 	remove_proc_entry("sched_debug", NULL);
+ 
+-	if (!proc_create_seq("sched_debug", 0444, NULL, &__orig_sched_debug_sops))
++	if (!proc_create("sched_debug", 0444, NULL, &__orig_sched_debug_fops))
+ 		return -ENOMEM;
+ 
+ 	return 0;
+@@ -338,14 +368,14 @@ int restore_sched_debug_procfs(void)
+ #endif
+ 
+ #ifdef CONFIG_SCHEDSTATS
+-extern struct seq_operations __orig_schedstat_sops;
++extern struct file_operations __orig_proc_schedstat_operations;
+ 
+ /* schedstat interface in proc */
+ int install_proc_schedstat(void)
+ {
+ 	remove_proc_entry("schedstat", NULL);
+ 
+-	if (!proc_create_seq("schedstat", 0444, NULL, &__mod_schedstat_sops))
++	if (!proc_create("schedstat", 0444, NULL, &__mod_proc_schedstat_operations))
+ 		return -ENOMEM;
+ 
+ 	return 0;
+@@ -355,7 +385,7 @@ int restore_proc_schedstat(void)
+ {
+ 	remove_proc_entry("schedstat", NULL);
+ 
+-	if (!proc_create_seq("schedstat", 0444, NULL, &__orig_schedstat_sops))
++	if (!proc_create("schedstat", 0444, NULL, &__orig_proc_schedstat_operations))
+ 		return -ENOMEM;
+ 
+ 	return 0;
+diff --git a/kernel/sched/mod/sched_rebuild.c b/kernel/sched/mod/sched_rebuild.c
+index 6e8a4c7..37427a8 100644
+--- a/kernel/sched/mod/sched_rebuild.c
++++ b/kernel/sched/mod/sched_rebuild.c
+@@ -6,8 +6,11 @@
+ #include <linux/sched.h>
+ #include "sched.h"
+ 
+-extern void __orig_set_rq_offline(struct rq*);
+-extern void __orig_set_rq_online(struct rq*);
++#define __orig_set_rq_online vmlinux_set_rq_online
++#define __orig_set_rq_offline vmlinux_set_rq_offline
++extern void __mod_set_rq_offline(struct rq*);
++extern void __mod_set_rq_online(struct rq*);
++
+ extern unsigned int process_id[];
+ 
+ extern struct sched_class __orig_stop_sched_class;
+@@ -40,6 +43,37 @@ struct sched_class *mod_class[] = {
+ #define NR_SCHED_CLASS 5
+ struct sched_class bak_class[NR_SCHED_CLASS];
+ 
++static void vmlinux_set_rq_online(struct rq *rq)
```

Comment:
How about handle these two functions like you do to `__orig_unregister_sched_domain_sysctl`?
Instead of renaming them, just define them as `__orig_unregister_sched_domain_sysctl`?